gstcefsrc.cc

```diff
@@ -830,6 +844,12 @@ gst_cef_src_class_init (GstCefSrcClass * klass)
           "(Example: --noexpose_wasm --expose-gc)",
           NULL, (GParamFlags) (G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS | GST_PARAM_MUTABLE_READY)));
 
+  g_object_class_install_property (gobject_class, PROP_LOG_SEVERITY,
+    g_param_spec_string ("log-severity", "log-severity",
```

Comment:
You want to register an enum here, and install the property as `g_param_spec_enum` :)

Reply:
Nice, that's much neater. I should've known there was gobject way!